lib/util/promises.ts

```diff
@@ -0,0 +1,3 @@
+export function toPromise<T>(value: T): Promise<T> {
+  return new Promise<T>((resolve) => resolve(value));
```

Comment:
```suggestion
  return Promise.resolve(value);
```

Isn't this better practise? Can't we simlpy inline `Promise.resolve(value)` ?

Reply:
Yeah. I'll get rid of this helper. You're right, it would only be useful if we were doing something more substantial in here like caching, or using `reject`.